controllers/auth.js

```diff
@@ -52,6 +52,17 @@ const githubAuth = (req, res, next) => {
   }
 };
 
+const signout = (req, res) => {
+  const cookieName = config.get("userToken.cookieName");
+  res.clearCookie(cookieName, {
+    domain: ".realdevsquad.com",
```

Comment:
As per [Express documentation](https://expressjs.com/en/api.html#res.clearCookie) I think we'll need to add all the options which are being passed while setting the cookie

Reply:
Yes samarpan, I went through the documentation, also the default path is '/' itself so I did not add it there. I added the domain as it creates an issue by taking the subdomain as the main domain.

Reply:
while the other parameters are the same as the main domain, so they needn't be set.

Reply:
What about httpOnly, secure and sameSite options? We're adding them while setting the cookie

Reply:
As per the recent commit this conversation is resolved.